index.html

```diff
@@ -0,0 +1,174 @@
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main id="id">
+      <section class="headline">
+        <div class="our-terms">
+          <div class="container">
+            <p class="slogan">" Every Talent Deserves a Chance."</p>
+            <div class="main-tilte">
+              <h1>All Companies Deserves Latest Tech in 2022</h1>
+            </div>
+          </div>
+        </div>
+        <div class="detail-container">
+          <div class="bg-color"></div>
+          <div class="main-content container flex-center">
+            <div class="detail container">
+              <p>
+                Technology is not simply hardware and software. It's also about
+                humans!
+              </p>
+            </div>
+            <div class="work-time container">
+              <span>24H/7D</span>
+              <span>Latest Routing Technology By Ayoub</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="mission" class="mission">
+        <div class="bg-color-mission"></div>
+        <div class="main-container-mission">
+          <h2 class="title">OUR MISSION</h2>
+          <ul class="mission-list">
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="flex-center content-mission">
+                <div class="icon flex-center">
+                  <img src="./img/clock-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Efficiency</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>Rapidly Giving the Best</p>
+                </div>
+              </div>
+            </li>
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="content-mission flex-center">
+                <div class="icon">
+                  <img src="./img/battery-full-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Maintainability</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>Provide the best After Sales Service</p>
+                </div>
+              </div>
+            </li>
+            <li class="mission-card container">
+              <div class="mission-card-bg"></div>
+              <div class="content-mission flex-center">
+                <div class="icon">
+                  <img src="./img/check-circle-solid.svg" alt="" />
+                </div>
+                <div class="mission-title">
+                  <h2>Predictability</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>
+                    Our devices know when they will be down, and they make you
+                    know beforehand!
+                  </p>
+                </div>
+              </div>
```

Comment:
- [ ] This section is supposed to have elements displayed as rows on the desktop view and as columns on the mobile one. Currently, they are only being displayed as columns. Please make sure to address this issue, I suggest you also use Flexbox to position the content of the single card, not just the cards themselves. This way, you can choose whether to display as rows or columns both the cards and their own content.

Reply:
Done